src/org/omegat/Bundle.properties

```diff
@@ -502,13 +502,13 @@ MW_OPTIONMENU_APPEARANCE_THEME_LABEL=Theme:
 MW_REOPEN_QUESTION=OmegaT needs to reload the project to take the modifications into account.\n\
 Reload the project now?
 
-TF_LOAD_WARN_SOURCE_LOOP_EXCEPTION=Skipped some source file(s)\n\
+TF_LOAD_WARN_SOURCE_LOOP_EXCEPTION=Skipped one or more source files\n\
   because the specified source folder contains a symbolic or hard link\n\
   that points back to itself or one of its subfolders.\n\
   Make sure all links are outside the source folder and subfolders.\n\
-  Please check a link: {0}
-TF_LOAD_WARN_FILE_ACCESS=Skipped some source file(s) because of access permission \
-  error in the source folder. Please check a file: {0}
+  Invalid link: {0}
+TF_LOAD_WARN_FILE_ACCESS=Skipped one or more source files due to an access permission \
+  error in the source folder. Skipped file(s): {0}
```

Comment:
The exception is raised when OmegaT visit problematic file, and report the problematic file.
Please assume there is  file A B C and P, and P is problematic. An order of visit is NOT an order of name, depends on OS.

case 1) 
walk  A ,  B,  P,  C  =>  OmegaT read A and B then raise error with file P.

case 2)
walk A, P, B, C => OmegaT read A then raise error with file P.

case 3)
walk P A B C => OmegaT read nothing and raise error with file P.

when error message said "Skipped file(s): P"   it is incorrect, because OmegaT actually skip file A B C in case 3, and OmegaT has no way  to know which files are skipped. OmegaT only knows files which are already read.

Reply:
This one had slipped under my radar.

So, in your examples, the `{0}` in the error message would expanded with the name of file P?

In that case:
~~Skipped files(s):~~ --> Unable to access: {0}

If there is there is more than one problematic file (files P1 and P2), will OmegaT pick up both or will it stop at the first problematic file it encounters?

Reply:
Thanks for the explanation.

I had a better idea for the message and just pushed a change. Let me know if that works.